xapian-letor/tests/api_letor.cc

```diff
@@ -104,26 +98,19 @@ gen_uniqterms_gt_doclen_db(Xapian::WritableDatabase& db, const string&)
     db.add_document(doc);
 }
 
-DEFINE_TESTCASE(ranker1, generated) {
+DEFINE_TESTCASE(listnet_ranker, generated) {
     Xapian::ListNETRanker ranker;
     string a = get_database_path("apitest_ranker1",
-				       gen_uniqterms_gt_doclen_db);
+				       populate_db);
     Xapian::Enquire enquire((Xapian::Database(a)));
     enquire.set_query(Xapian::Query("lions"));
     Xapian::MSet mymset = enquire.get_mset(0, 10);
-    string query ="20001 'lions'";
-    string qrel = "20001 Q0 1 1\n20001 Q0 2 2";
-    std::ofstream myfile;
-    myfile.open("query.txt");
-    myfile << query;
-    myfile.close();
-    myfile.open("qrel.txt");
-    myfile << qrel;
-    myfile.close();
-    Xapian::prepare_training_file(a, "query.txt", "qrel.txt", 10);
+    Xapian::prepare_training_file(a, "testdata/query.txt",
+				  "testdata/qrel.txt", 10,
+				  "training-data.txt");
```

Comment:
so should I delete the files at the end of the tests ?? .... because Ranker does need the training-data.txt to train the model